features/src/main/feature/feature.xml

```diff
@@ -29,13 +29,14 @@
         <feature>openhab-core-model-item</feature>
         <feature>openhab-core-model-sitemap</feature>
         <feature>openhab-core-ui-icon</feature>
-        <feature>openhab-core-ui</feature>
-        <feature dependency="true">openhab.tp-jaxb</feature>
+	<feature>openhab-core-ui</feature>
```

Comment:
here's still some indentation issues (tabs instead of spaces)

Reply:
It will be reformatted by https://github.com/openhab/openhab-webui/pull/240 anyways. ;-)

Reply:
ok, then let's ignore it here...